### PR TITLE
Upgrade to readme

### PR DESCRIPTION
**Pull request by AndrewJesse**
The revised README.md enhances user experience by providing a detailed introduction, quick start guide, and clearer sections for key features, contribution and resources, along with an improved licensing description. It also includes more direct documentation links for better accessibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,47 @@
...
-- Authorization
-- Logging
-- Events and Cron Jobs
-- Mail
-- CLI Commands
+## Requirements
+Before creating your first spark project, you should ensure that your local machine has:
+- [.NET 7](https://dotnet.microsoft.com/download)
+- [Entity Framework Core tools](https://learn.microsoft.com/en-us/ef/core/cli/)
+- [Node JS](https://nodejs.org/en/download) "For executing styles provided by Tailwind CSS"
```
Comment (westonwalker) [inline README.md:10]: Please remove this line. NodeJS is no longer needed.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,47 @@
...
+spark new MyApp
+```
+That's it! Now dive into our comprehensive [documentation](https://blazorspark.com/docs/introduction/) for next steps and deeper insights.
 
-## License
-The Spark framework is open-sourced software licenced under the MIT license.
\ No newline at end of file
+# Documentation
+
+Blazor Spark has thorough [documentation](https://blazorspark.com/docs/introduction/), making it easy to get started. It's also built on top of [.NET Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor), so if you're familiar with it, you'll find Spark incredibly intuitive to use.
```
Comment (westonwalker) [inline README.md:24]: Replace all occurrences of .NET Blazor with Blazor Server.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,47 @@
...
-- Mail
-- CLI Commands
+## Requirements
+Before creating your first spark project, you should ensure that your local machine has:
+- [.NET 7](https://dotnet.microsoft.com/download)
+- [Entity Framework Core tools](https://learn.microsoft.com/en-us/ef/core/cli/)
+- [Node JS](https://nodejs.org/en/download) "For executing styles provided by Tailwind CSS" 
 
-## Learning Blazor Spark
-Spark has thorough documentation making it simple to get started with. It's also built on top of .NET Blazor. So if your familiar with it getting started with Spark will be easy.
```
Comment (westonwalker) [inline README.md:16]: Replace all occurrences of .NET Blazor with Blazor Server.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,47 @@
+# Blazor Spark - The .NET Blazor Framework for Makers
 ![Blazor Spark Logo](https://i.imgur.com/kqd8VIg.png "Blazor Spark")
 
-## About Blazor Spark
-Spark is a web application framework built on top of .NET Blazor. It takes the pain out of developing .NET Blazor apps by providing a working full stack application and providing out of the box solutions for common tasks such as:
+Welcome to Blazor Spark, a robust, open-source web application framework built on .NET Blazor, designed for makers like you. With Spark, you can build production-grade, full-stack web applications without breaking a sweat.
```
Comment (westonwalker) [inline README.md:4]: Replace all occurrences of .NET Blazor with Blazor Server.

**Review by westonwalker — CHANGES_REQUESTED** [review]
This looks great :) Just a couple of small verbiage changes and I'll approve.

File: README.md (hunk 1 above)
Comment (AndrewJesse) [reply README.md]: Got it removed.

File: README.md (hunk 2 above)
Comment (AndrewJesse) [reply README.md]: Replaced all occurences

**Review by westonwalker — APPROVED** [review]
Approved. Thanks for the contribution Andrew!

**Comment by AndrewJesse** [comment]
My pleasure!

**Pull request merged**